faros_event.sh

```diff
@@ -36,9 +36,10 @@ build_statuses=$(printf '%s\n' "$(IFS=,; printf '%s' "${BUILD_STATUSES[*]}")")
 declare -a DEPLOY_STATUSES=("Success" "Failed" "Canceled" "Queued" "Running" "RolledBack" "Custom")
 deploy_statuses=$(printf '%s\n' "$(IFS=,; printf '%s' "${DEPLOY_STATUSES[*]}")")
 
+# Script settings' defaults
 dry_run=${FAROS_DRY_RUN:-0}
-silent=0
-debug=0
+silent=${FAROS_SILENT:-0}
+debug=${FAROS_DEBUG:-0}
```

Comment:
These settings, in my mind, are more fundamental to how they affect the script where as `--no_build_object` is more external and tied to the models the script produces. That is why I have them separated out.